content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/create-slack-alerts/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Create Slack Alerts from Enforce Cloud Events"
```

Comment:
so i know the other tutorial is already published, but shouldn't this title be "Create Slack Alerts from Enforce CloudEvents"? i.e., no space between cloud and events?

Reply:
Oh yeah, not sure how I managed to do that, will update the other one too